### PR TITLE
Fixed an anchor and corrected a typo

### PR DESCRIPTION
**Pull request by neotan**
Just made some minor updates.

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Hmm, looks like this is changing the generated HTML instead of the source (index.bs). No big deal, but we will need to back-port these to index.bs - as these changes will get lost next time the spec is generated.

**Comment by miguelao** [comment]
Yeah, @neotan, please modify the `index.bs` file instead and then run e.g.
`curl https://api.csswg.org/bikeshed/ -F file=@index.bs  > index.html`
and/or
`curl https://api.csswg.org/bikeshed/ -F file=@index.bs -F output=err  && echo`
to see the (potential) errors.

File: index.bs [hunk 1]
```diff
@@ -129,7 +129,7 @@ Example implementations of face detection are e.g. <a href="https://developer.an
...
+{{BarcodeDetector}} represents an underlying accelerated platform's component for detection in images of QR codes or barcodes.  It provides a single {{BarcodeDetector/detect()}} operation on an {{ImageBitmapSource}} which result is a Promise.  This method MUST reject this Promise in the cases detailed in [[#image-sources-for-detection]]; otherwise it MAY queue a task using the OS/Platform resources to resolve the Promise with a sequence of {{DetectedBarcode}}s, each one essentially consisting on and delimited by a {{DetectedBarcode/boundingBox}} and a series or {{Point2D}}s, and possibly a {{DetectedBarcode/rawValue}} decoded {{DOMString}}.
```
Comment (miguelao) [inline index.bs]: The changes themselves are OK, please run
`curl https://api.csswg.org/bikeshed/ -F file=@index.bs  > index.html`
and upload the compiled `.html` file as well.

(Having to compile and upload both is kind of lame, but in low traffic repos like this one, maintaining travis jobs is a bit of an overkill, hence the manual compiling).

Also I suspect #22 is not needed if we continue this one, correct?

Comment (neotan) [inline index.bs]: Hi, @miguelao , let's use #22 as final pr, I submit Chinese .bs and .html. sorry for requesting 2 pr separately.

Comment (miguelao) [inline index.bs]: Ok, IIUC, let's use this one for the small corrections, and leave #22 (of which you still
need to update the title) for the Chinese translation part.  Here, you still have to run
`curl https://api.csswg.org/bikeshed/ -F file=@index.bs > index.html`
and upload the compiled .html file as well :-)

**Pull request merged**